crates/starknet-devnet-types/src/serde_helpers.rs

```diff
@@ -103,7 +102,7 @@ pub mod hex_string {
     where
         S: Serializer,
     {
-        s.serialize_str(contract_address.to_prefixed_hex_str().as_str())
+        serialize_patricia_key_to_prefixed_hex(&contract_address.0, s)
```

Comment:
Does this take care of the address format in `devnet_getPredeployedAccounts`?

Reply:
yes